### DISCUSSION TITLE
Deceptive site ahead on Git page

### DISCUSSION THREAD

**Discussion by Sidu52** [community/community#51769] [Actions]
### Select Topic Area

Question

### Body

When i open project by git page URL. He show Dangerous sing.

Attackers on sidu52.github.io may trick you into doing something dangerous like installing software or revealing your personal information (for example, passwords, phone numbers, or credit cards). 

how can i fix it

**Comment by meppu** [comment]
Any of your sites had something that looks like some other website? For example a twitter clone for practicing. Google may mark your site as phishing website if so.


![image](https://user-images.githubusercontent.com/112294217/229371211-acee56d4-6b15-4990-9f9d-67c02ea05e19.png)


Visit [google safe browsing documentation](https://developers.google.com/search/docs/advanced/security/social-engineering?hl=en) for more information.

You can also report incorrect phishing warning [here](https://safebrowsing.google.com/safebrowsing/report_error/?hl=en). since it is browser-related, github support can't help you.
